### DISCUSSION TITLE
Many Question about sceneform-android

### DISCUSSION THREAD

**Discussion by Pey** [sceneview/sceneform-android#442] [Q&A]
Hi,
veryThank you for this nice source.
I have many question about  sceneform-android so i am sorry for it.
1- how to load 3d object from specific folder on the SD card in an Android application ?

2- Which format support in sceneform-android instead of .glb ?

3- Can i use fbx or .obj ? if your answer Yes, How can i do it?

4- how many users can use from application that have sceneform-android? 

5- how to put arrow image ( up, down, left, right ) on screen to user used them to change 3d object position?

6- What is the minimum and maximum size of 3d object that  sceneform-android supports?

**Comment by Pey** [comment]
please answer me.thanks

**Comment by grassydragon** [comment]
Hi!

1\. Make sure the app has access to the storage location you want to load a model from (https://developer.android.com/guide/topics/data) and use a file URI to load a model.
2-3\. No, you can only use the glTF format.
4\. This question isn't very clear.
5\. You can place buttons over the `ArFragment` in your layout.
6\. These limits are hard to define, however, an object should have a reasonable size for the best user experience.

**Comment by Pey** [comment]
Thanks for your responses.
1.I can't load 3d object (.glb) from specific path from folder in sd card.

**Comment by Pey** [comment]
How to can create AR Indoor Navigation and Outdoor Navigation using sceneform ?or ARCore ?
Please answer me ,this project very important and Attractive for me

**Comment by Pey** [comment]
please answer me 
thank you

**Comment by grassydragon** [comment]
You can see these two awesome projects created by our community members (however, these projects use Sceneview instead of Sceneform):
- https://github.com/Gebort/FESTU.Navigator
- https://github.com/morhenny/ar-localization
